src/components/Header.vue

```diff
@@ -34,13 +34,13 @@
 
 <script lang="ts">
 import { defineComponent } from 'vue'
-import { useStep } from '../store/step'
+import { useAppState } from '../store/app-state'
```

Comment:
Please remove this because it is not used.

Reply:
done, removed the `setup` method entirely